adapters/revcontent/revcontenttest/params/race/banner.json

```diff
@@ -0,0 +1,3 @@
+{
+}
```

Comment:
If your adapter doesn't support any parameters then you don't need these params files. You can remove these.